app/templates/_index.html

```diff
@@ -12,7 +12,6 @@
   <script src="js/phaser.js"></script>
   <script src="js/game.js"></script>
 
-
-
+  <script src="//localhost:35729/livereload.js"></script>
```

Comment:
Do you mean this port, or the port that the local server is served on? (port 9000). 

I see an advantage to being able to configure both, but the existing code used a fixed port, so this extended that. In my opinion, raise an issue to say "add feature to configure ports" so the pull requests are kept relevant.